neural_network/lstm_stock/core/data_processor.py

```diff
@@ -0,0 +1,84 @@
+import math
+import numpy as np
+import pandas as pd
+
+class DataLoader():
+    """A class for loading and transforming data for the lstm model"""
+
+    def __init__(self, filename, split, cols):
+        dataframe = pd.read_csv(filename)
+        i_split = int(len(dataframe) * split)
+        self.data_train = dataframe.get(cols).values[:i_split]
+        self.data_test  = dataframe.get(cols).values[i_split:]
+        self.len_train  = len(self.data_train)
+        self.len_test   = len(self.data_test)
+        self.len_train_windows = None
+
+    def get_test_data(self, seq_len, normalise):
+        '''
+        Create x, y test data windows
+        Warning: batch method, not generative, make sure you have enough memory to
+        load data, otherwise reduce size of the training split.
+        '''
+        data_windows = []
+        for i in range(self.len_test - seq_len):
+            data_windows.append(self.data_test[i:i+seq_len])
+
+        data_windows = np.array(data_windows).astype(float)
+        data_windows = self.normalise_windows(data_windows, single_window=False) if normalise else data_windows
```

Comment:
This line is too long for a ternary if...  Just do a normal if and wrap lines to 88 char max.